google-cloud-core/src/main/java/com/google/cloud/ServiceOptions.java

```diff
@@ -73,7 +73,7 @@
   private static final String MANIFEST_ARTIFACT_ID_KEY = "artifactId";
   private static final String MANIFEST_VERSION_KEY = "Implementation-Version";
   private static final String ARTIFACT_ID = "google-cloud-core";
-  private static final String LIBRARY_NAME = "gcloud-java";
```

Comment:
@garrettjonesgoogle We definitely want to leave UA alone for the discovery-based APIs like BQ, Storage, and Translate. We're not updating how these are handled server-side for now.

For the gRPC-based APIs, we can stop doing anything in UA; no one is depending on what we have been doing. It would actually be beneficial to do so for server-side performance, but this is not a hard requirement for this round of header updates.